### ISSUE TITLE
16.6 crashing on sony bravia

### ISSUE THREAD

**Issue by batkung** [koying/SPMC#837]
SPMC just updated last night via google playstore to 16.6 from 16.5.

Now I am seeing random crashes, sometimes when playing media from a NAS, sometimes when doing nothing.

has anyone else seen this?..it never happened for me on 16.5.

**Comment by bluppblupp** [comment]
Yeah, same here. Almost every time I close SPMC, I get "Unfortunately, SPMC has stopped" and the option to report or just press OK.

Running 16.6.0 on Sony Bravia 55X8505C.

**Comment by CLFNole** [comment]
I have the same thing but I think its more of a problem and/or conflict with the Android Marshmallow update we got on our TVs.  I have a Nexus Player in my bedroom running Nougat 7 and don't get the error at all.  We are supposed to get an update to Nougat this year so maybe that will fix things who knows?

**Comment by Orthanes** [comment]
I have the same problem on Sony Bravia. SPMC crashes randomly. I get the option to post a report which I do every time.

**Comment by lukehansford** [comment]
Same issue here on brand new 498088 Sony android. No error on 17 or 17.1.

On every exit I get the same message unfortunately had to close message and offers the option to press ok or report.

Please can we get this message removed or turned off. It closes down completley fine it's just annoying on every close display without any reason. Shield owners have reported it happening to them also so it's not to do with just Sony.

**Comment by batkung** [comment]
blockquote, div.yahoo_quoted { margin-left: 0 !important; border-left:1px #715FFA solid !important; padding-left:1ex !important; background-color:white !important; } It's not just on close, it seems random on my setup.


Sent from Yahoo Mail for iPhone


On Wednesday, March 22, 2017, 19:13, lukehansford <notifications@github.com> wrote:


Same issue here on brand new 498088 Sony android. No error on 17 or 17.1.

On every exit I get the same message unfortunately had to close message and offers the option to press ok or report.

Please can we get this message removed or turned off. It closes down completley fine it's just annoying on every close display without any reason. Shield owners have reported it happening to them also so it's not to do with just Sony.

**Comment by CLFNole** [comment]
Mine no longer gives me the option to report it just has OK now.

**Comment by CLFNole** [comment]
Someone in another thread mentioned removing Sports Devil solved his crashing problem so we all might want to test that out.

**Comment by barenaked** [comment]
My W750c just updated to 16.6.0 and it's randomly crashing now, too, even during playback. I don't have any plugins installed except artwork downloader and the usual stuff that comes with official skins. 

My wife is really annoyed :/

Any news why 16.6.0 crashes? 

It's not the marshmallow update because 16.5.5 worked just fine.

**Comment by stevieland58** [comment]
It's definitely SportsDevil causing the issue, was stalling Kodi from closing before its last update.

Just go into addons and then system then running and you'll see it's running when it shouldn't be, disable it then close the whole app and you'll see that the error is gone!

Hopefully those at SportsDevil fix it soon, it's not a SPMC or KODI or any another forks issue...

**Comment by CLFNole** [comment]
I completely removed Sports Devil and still get the error message after closing.

**Comment by stevieland58** [comment]
Including its dependencies and addon data? 
Anyways work for me on my Sony android TV and other devices! 
The error started from the 20th of match SportsDevil update, never before wnd once disabled from beginning has ever happened again!

**Comment by bluppblupp** [comment]
I factory restored the tv and reinstalled SPMC, and I still have the same problem unfortunately.

**Comment by stevieland58** [comment]
Did you do a fresh SPMC install or restore from backup?
It's strange, for me if was a sports devil dependency not closing down correctly using SPMC and while my tv was running on lollipop as I have tried it with Kodi and CEMC on the same tv and no issue at all.
I also factory reset the tv and  now the same thing is happening but only using SPMC and only after it upgraded to marshmallow.
Now what is it though? 
Is it a marshmallow issue with SPMC? I'm running 16.7 side loaded and still happening.
Try CEMC. Doesn't happen at all and they python is the lates for streaming.

**Comment by stevieland58** [comment]
Ok I figured it out. its the Confluence skin causing it to crash.

I first deleted all the orphaned dependencies and it made things faster but still had the error message, then changed the skin and no more message...

Has been no error for a few days running now so seems that's the issue...

You're welcome.
